app/views/categories/show.html.erb

```diff
@@ -0,0 +1,56 @@
+<div>
+
+  <% ### Back Button when no child category exists ### %>
+  <% if categories.count < 1 %>
+    <p><%= link_to "Back", :back, class: "waves-effect waves-light btn" %></p>
+  <% end %>
+
+  <% ### When either a child category or note is present %>
+  <% if (categories.count >= 1) || (notes.count >= 1) || (photos.count >= 1)%>
```

Comment:
this is just a style thing, but it might make it a bit more readable. `unless (categories.empty?) && (notes.empty?) && (photos.empty?)`